CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
-# 2.2.0
+# 2.3.0
+
+### :hammer_and_wrench: Improvements
+* Add `httpClientBuilder` attribute to `QldbDriverBuilder` to allow customized HTTP client.
```

Comment:
nit: `to support use of custom HTTP clients.` sounds better.

Reply:
Updated.